great_expectations/rule_based_profiler/data_assistant/data_assistant.py

```diff
@@ -76,7 +76,7 @@ class MetaDataAssistant(ABCMeta):
 
     def __new__(cls, clsname, bases, attrs):
         """
-        Instantiate class as part of descentants calling "__init__()" and register its type in "DataAssistant" registry.
+        Instantiate class as part of descendants calling "__init__()" and register its type in "DataAssistant" registry.
```

Comment:
💙  -- thank you (my mistake)!

Reply:
No problem. I have issues with spelling too. I installed a spell checker extension on VSCode. I'm not sure if pycharm has a similar feature.